R/install.r

```diff
@@ -47,6 +47,8 @@
 #'   It defaults to the option \code{"Ncpus"} or \code{1} if unset.
 #' @param force_deps whether to force installation of dependencies even if their
 #'   SHA1 reference hasn't changed from the currently installed version.
+#' @param metadata \code{list} of metadata (normally generated by
+#'   \code{remote_metadata}) to add to the DESCRIPTION file after installation.
```

Comment:
I'd say: `Named list of metadata entries to be added to the \code{DESCRIPTION} prior to installation.`